doc/MCP/FTS_Implementation_Plan.md

```
@@ -0,0 +1,582 @@
+  metadata TEXT                -- JSON with original row data
+);
+
+-- FTS5 virtual table (external content)
+CREATE VIRTUAL TABLE fts_search_<sanitized_schema>_<sanitized_table> USING fts5(
+  content,
+  metadata,
+  content='fts_data_<sanitized_schema>_<sanitized_table>',
+  content_rowid='rowid',
+  tokenize='porter unicode61'
+);
+
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)